### ISSUE TITLE
scrollbar too dark

### ISSUE THREAD

**Issue by drakkan** [pop-os/gtk-theme#66]
I love this theme, the only thing that I don't like is the scrollbar in app like thunderbird or firefox, it seems too big and too dark, please take a look at the attached screenshot. 

In application like nautilus the scrollbar is thinner and it disappers after a short while, but in applications like thunderbird it seems too big and too dark, maybe a taste thing

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
**Reporter info**

```
 * Distribution - archlinux
 * Gtk+ 3.0 version - 3.22.24
 * Desktop environment - Gnome
 * Pop version - Pop!_GTK+ Version 2.2.0r0
 ```

![screen](https://user-images.githubusercontent.com/553263/31820246-c6d9b092-b5a0-11e7-9463-b7174a2b4302.png)

**Comment by drakkan** [comment]
the scroolbar in gtk2 apps seems better to me, please take a look at this screenshot

![gtk2_screen](https://user-images.githubusercontent.com/553263/31824801-70978820-b5b0-11e7-918f-decd8602b73e.png)

**Comment by isantop** [comment]
The scrollbar styling is something that I intend to look into further. One large, complicating factor is that the Terminal requires a dark scrollbar in order for it to be visible. Unless the theme is switched to Dark, the terminal uses the same scrollbars as all other light-theme apps, which uses a light colored tray and a slightly darker actual scrollbar widget. However, the tray itself is semi-transparent, and this makes it very dark in the terminal. In order to keep the scrollbar visible in the terminal without switching to a light-colored one (which would be invisible in every other app), in needs to be very dark.

**Comment by isantop** [comment]
Fixed in 2.2.3 ac0cf452285d3d9045360f674bfbba5e4aa46acf
